crates/sui-benchmark/src/bin/stress.rs

```diff
@@ -274,39 +264,19 @@ async fn main() -> Result<()> {
             });
             Arc::new(configs)
         };
-        let gateway_config = GatewayConfig {
-            epoch: 0,
-            validator_set: configs.validator_set().to_vec(),
-            send_timeout: Duration::from_secs(4),
-            recv_timeout: Duration::from_secs(4),
```

Comment:
^

Reply:
If there is no clue for why this is causing more timeout errors, let's just try setting this to 30s.